src/frontend/src/routes/router.tsx

```diff
@@ -37,6 +37,7 @@ const Router = () => {
         <Route path="/pia-intake" element={<PIAIntakeFormPage />} />
         <Route path="/pia-result" element={<PIAIntakeResultsPage />} />
         <Route path="/pia-list" element={<PIAList />} />
+        <Route path="/pia/result/:id" element={<PIAList />} />
```

Comment:
If the view page does not exist, no point keeping it to `PIAList`. Please add it when we have the view page ready.